src/library_browser.js

```diff
@@ -1022,7 +1022,7 @@ var LibraryBrowser = {
       // to queue the newest produced audio samples.
       // TODO: Consider adding pre- and post- rAF callbacks so that GL.newRenderingFrameStarted() and SDL.audio.queueNewAudioData()
       //       do not need to be hardcoded into this function, but can be more generic.
-      if (typeof SDL == 'object' && SDL.audio && SDL.audio.queueNewAudioData) SDL.audio.queueNewAudioData();
+      if (typeof SDL == 'object') SDL.audio?.queueNewAudioData?.();
```

Comment:
Does `?.` work on checking if a symbol exists in the global scope? If so we could do
```suggestion
      SDL?.audio?.queueNewAudioData?.();
```
Perhaps that would run into a closure error though...